wandb/sdk/launch/builder/kaniko_builder.py

```diff
@@ -151,13 +154,15 @@ def from_config(
         build_job_name = config.get("build-job-name", "wandb-launch-container-build")
         secret_name = config.get("secret-name", "")
         secret_key = config.get("secret-key", "")
+        image = config.get("image", "gcr.io/kaniko-project/executor:1.11.0")
```

Comment:
Can we rename this `kaniko_image` or something? I worry about the number of nested configs called image we already have

Reply:
yeee